connection.py

```diff
@@ -0,0 +1,12 @@
+class Connection():
```

Comment:
Needs documentation. Also, I think you should drop `()`. You could replace with `(object)` since you're using python2 and probably want a pure object.